oxidation/libparsec_python/src/protocol/block.rs

```diff
@@ -106,7 +107,9 @@ impl BlockCreateRep {
 
     #[classmethod]
     fn load(_cls: &PyType, buf: Vec<u8>) -> PyResult<Self> {
-        Ok(Self(block_create::Rep::load(&buf)))
+        block_create::Rep::load(&buf)
+            .map(Self)
+            .map_err(|e| PyErr::new::<PyValueError, _>(e.to_string()))
```

Comment:
You can use directly PyValueError::new instead

Reply:
😯

Reply:
done